config/locales/en_BE.yml

```diff
@@ -1522,6 +1522,7 @@ en_BE:
   orders_changeable_orders_alert_html: This order has been confirmed, but you can make changes until <strong>%{oc_close}</strong>.
   products_clear_all: Clear all
   products_showing: "Showing:"
+  products_or: "OR"
```

Comment:
Also because of the way we use Transifex, we only update translation keys in `en.yml`, and don't commit any changes to the other locale files as they are updated separately.

Reply:
Could you revert the changes in `en_BE.yml`, `en_CA.yml`, `en_US.yml`, and `en_GB.yml`? Thanks! :heart: